src/lib/models/workflow-status.ts

```diff
@@ -0,0 +1,19 @@
+import type { WorkflowStatus } from '$lib/types/workflows';
+
+export type WorkflowStatuses = readonly WorkflowStatus[];
+export type WorkflowFilters = readonly (WorkflowStatus | 'All')[];
+
+export const workflowStatuses: WorkflowStatuses = [
+  'Running',
+  'TimedOut',
+  'Completed',
+  'Failed',
+  'ContinuedAsNew',
+  'Canceled',
+  'Terminated',
+] as const;
+
+export const workflowStatusFilters: WorkflowFilters = [
+  ...workflowStatuses,
+  'All',
+] as const;
```

Comment:
Looks like `All` got moved to the end of the list ⬇️ 
<img width="309" alt="Screenshot 2023-05-30 at 3 09 30 PM" src="https://github.com/temporalio/ui/assets/15069288/32b9c64e-334e-46e0-b0e8-d0ee3caf6acd">
and we probably still want it at the top

```suggestion
export const workflowStatusFilters: WorkflowFilters = [
  'All',
  ...workflowStatuses,
] as const;
```